README.md

```diff
@@ -42,7 +42,9 @@ Flags:
       --pipeline.encode.type string            Encode type: prom, none (default "none")  
       --pipeline.extract.aggregates string     Aggregates (see docs)  
       --pipeline.extract.type string           Extract type: aggregates, none (default "none")  
+      --pipeline.ingest.collector string       Ingest collector API  
       --pipeline.ingest.file.filename string   Ingest filename (file)  
+      --pipeline.ingest.kafka string           Kafka ingest API  
```

Comment:
I think that there is a place to add an explanation on Kafka ingest as a small section in the ream-me  --- as alternative if we want to look forward ... you can extend https://github.com/netobserv/flowlogs2metrics/pull/55/files#diff-67a6e36014be865053c43d38b2091bd2d3f17a91b117d97c0f5244cd7d71127dR29 to include all the info ... going forward my thinking was to add less and less text to the README and move the text to be part of the automatic generated code

Reply:
Added section to api.md